### ISSUE TITLE
Prices

### ISSUE THREAD

**Issue by bokivaske** [renzuzu/renzu_clothes#20]
Hello, is there any easy way to put a fixed price, for example, that all clothes cost $ 100

**Comment by Antarcy** [comment]
up

**Comment by Antho7Twitch** [comment]
Nope. you have the components.lua with all the prices, i suppose it needs to be configured manually :(

**Comment by JohnnyS** [comment]
> 

Good find I was checking to see if I can find something. Thank god dont have to edit 9999999 things LOL
